src/gesture-recognizers.js

```diff
@@ -774,7 +793,11 @@ export class PinchRecognizer extends GestureRecognizer {
 
   /** @override */
   onTouchEnd(e) {
-    this.end_(e);
+    // Number of current touches on the page
+    const {touches} = e;
+    if (touches.length <= 1) {
```

Comment:
The number 1 feels a bit weird. Feels like this might make more sense as `if (touches.length < 2)`, since 2 is the number of touches this gesture is interested in, so it seems to be more natural.

Reply:
Done